### PR TITLE
spring roo 1.3.2

### PR DESCRIPTION
**Pull request by arahman**
update to spring roo 1.3.2

### CODE REVIEW

File: Library/Formula/spring-roo.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class SpringRoo < Formula
   desc "Rapid application development tool for Java developers"
   homepage "http://www.springsource.org/spring-roo"
-  url "https://s3.amazonaws.com/spring-roo-repository.springsource.org/release/ROO/spring-roo-1.3.1.RELEASE.zip"
-  version "1.3.1"
-  sha256 "50be1c39ee56a0f2fba0109ed9326fbab018e08f9377dca3202fc8f8a4b5a784"
+  url "http://spring-roo-repository.springsource.org.s3.amazonaws.com/release/ROO/spring-roo-1.3.2.RELEASE.zip"
```
Comment (dunn) [inline Library/Formula/spring-roo.rb]: Looks like the 
```
https://s3.amazonaws.com/spring-roo-repository.springsource.org/release/ROO/spring-roo-1.3.2.RELEASE.zip
```
style still works, and that will let us stick with HTTPS. I get an invalid certificate error with `https://spring-roo-repository.springsource.org.s3.amazonaws.com/release/ROO/spring-roo-1.3.2.RELEASE.zi
p`

**Comment by arahman** [comment]
thanks for the feedback, I've updated the URL.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/5ca3ebd33f21ff424c3c91ab25f7c79f09f0cc3d. Thank you for your contribution to Homebrew @arahman; we appreciate it! :smiley_cat:

**Comment by arahman** [comment]
:)

**Pull request closed without merging**